daemon/api.go

```diff
@@ -256,6 +256,7 @@ func sysInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 		"os-release":     release.ReleaseInfo,
 		"on-classic":     release.OnClassic,
 		"managed":        len(users) > 0,
+		"forced-devmode": release.ReleaseInfo.ForceDevMode(),
```

Comment:
OK, will rework this and will introduce a new `debug` command for the `snap` binary.